src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -28,7 +28,17 @@ Use "monitor" to run a program on a attached monitor.
 View all attached peripherals with "peripherals".
 Use "time" to see the in-game time.
 You can set the label of a computer with "label set".
-A computer needs a label to keep its files if it got destroyed.
+A computer needs a label to keep its files if it gets destroyed.
 You can disable auto completion in the shell with "set shell.autocomplete false".
 You can disable auto completion in edit with "set edit.autocomplete false".
 Feeling creative? Use a printer to print a book!
+Files beginning with a "." character are hidden from "list" by default.
+Running "set" lists all currently set settings.
+Some programs are only available on advanced computers, turtles, pocket computers, or command computers.
+The "equip" and "unequip" programs let you add or remove peripherals from a turtle or pocket computer without crafting.
+You can change the color of a disk by crafting it with dye.
+Right-clicking a turtle with a dye changes its color.
+Distances from ender/wireless modems are relative to the full-block modems or computers that they are placed on.
```

Comment:
Remove this one - it's better suited for the wiki/help files about GPS.

Reply:
It turns out that the [new wiki](https://wiki.computercraft.cc/GPS_Hosts) actually has nothing on GPS hosts as far as I can tell (it looks like search is broken?) so I may have to write some basic documentation for that.